cmd2/parsing.py

```diff
@@ -29,46 +29,41 @@ class Statement(str):
     :var raw:               string containing exactly what we input by the user
     :type raw:              str
     :var command:           the command, i.e. the first whitespace delimited word
-    :type command:          str or None
+    :type command:          str
     :var multiline_command: if the command is a multiline command, the name of the
-                            command, otherwise None
-    :type command:          str or None
-    :var args:              the arguments to the command, not including any output
+                            command, otherwise empty
+    :type command:          str
+    :var arg_list:          list of arguments to the command, not including any output
                             redirection or terminators. quoted arguments remain
                             quoted.
-    :type args:             str or None
-    :var: argv:             a list of arguments a la sys.argv. Quotes, if any, are removed
```

Comment:
Ok, so it looks like you converted ``argv`` into a property, so it shouldn't have any effect on end users, but perhaps we have reduced storage of redundant data.